src/main/features/claimant-response/views/court-offered-set-date.njk

```diff
@@ -10,7 +10,7 @@
   <div class="grid-row">
     <div class="column-two-thirds">
       <p>{{ t('Based on financial details the defendant gave us, we don’t believe they can pay by your proposed date.') }}</p>
-      <p>{{ t('The court’s calculator has determined {{ paymentDate }} as the date when they can pay you in full.', { paymentDate: paymentDate | date }) }}</p>
+      <p>{{ t('The court’s calculator has determined {{ paymentDate }} as the date when they can pay you in full.', { paymentDate: date }) }}</p>
```

Comment:
You don't need `{ paymentDate: date | date }` filter?

Reply:
`date` is already defined and formatted earlier in the file